src/types.ts

```diff
@@ -187,19 +186,55 @@ export type OutputSelectorFields<
   MemoizeFunction extends UnknownMemoizer = typeof defaultMemoize,
   ArgsMemoizeFunction extends UnknownMemoizer = typeof defaultMemoize
 > = {
-  /** The final function passed to `createSelector`. Otherwise known as the `combiner`.*/
+  /**
+   * The final function passed to `createSelector`. Otherwise known as the `combiner`.
+   */
   resultFunc: Combiner<InputSelectors, Result>
-  /** The memoized version of {@linkcode OutputSelectorFields.resultFunc resultFunc}. */
+
+  /**
+   * The memoized version of {@linkcode OutputSelectorFields.resultFunc resultFunc}.
+   */
   memoizedResultFunc: Combiner<InputSelectors, Result> &
     ExtractMemoizerFields<MemoizeFunction>
-  /** Returns the last result calculated by the output selector. */
+
+  /**
+   * @Returns The last result calculated by {@linkcode OutputSelectorFields.memoizedResultFunc memoizedResultFunc}.
+   */
   lastResult: () => Result
-  /** An array of the input selectors. */
+
+  /**
+   * The array of the input selectors used by `createSelector` to compose the
+   * combiner ({@linkcode OutputSelectorFields.memoizedResultFunc memoizedResultFunc}).
+   */
   dependencies: InputSelectors
-  /** Counts the number of times the output has been recalculated. */
+
+  /**
+   * Counts the number of times {@linkcode OutputSelectorFields.memoizedResultFunc memoizedResultFunc} has been recalculated.
+   */
   recomputations: () => number
-  /** Resets the count of `recomputations` count to 0. */
+
+  /**
+   * Resets the count of {@linkcode OutputSelectorFields.recomputations recomputations} count to 0.
+   */
   resetRecomputations: () => 0
```

Comment:
:hand: Nitpicking a bit. I know these reset methods _do_ return `0` because of the shorthand logic, but _conceptually_ they really oughta be `() => void`

Reply:
Makes sense.

Reply:
do you think I should change the implementation to 
```ts
resetRecomputations: () => {
  recomputations = 0
}
```